api/api_sources/schema-migration-sql/WaterBodySchema/WaterBodySchema.sql

```diff
@@ -22,17 +18,13 @@ ALTER TABLE water_body ADD COLUMN distance NUMERIC(10, 5);
         
 COMMENT ON TABLE water_body IS 'The table to store all lake/water body information. Watercraft observation require  information regarding its source and destination.';
 COMMENT ON COLUMN water_body.water_body_id IS 'Auto generated primary key';
-COMMENT ON COLUMN water_body.type_enum IS 'Type of the water-body. This type is application specific enum values';
-COMMENT ON COLUMN water_body.name IS 'Common or popular name of the water-body';
-COMMENT ON COLUMN water_body.type_name IS 'Type of the water-body. This type is application specific string enum values. Like lake/river';
-COMMENT ON COLUMN water_body.type_code IS 'Type of the water-body. This type is application specific enum values.';
-COMMENT ON COLUMN water_body.water_body_location_latitude IS 'Latitude of treatment  location';
-COMMENT ON COLUMN water_body.water_body_location_longitude IS 'Longitude of treatment location';
-COMMENT ON COLUMN water_body.country IS 'Country of the water-body';
-COMMENT ON COLUMN water_body.province IS 'Province of the water-body location';
-COMMENT ON COLUMN water_body.abbrev IS 'Abbreviation of the water state/province';
-COMMENT ON COLUMN water_body.closest IS 'Closest landmark';
-COMMENT ON COLUMN water_body.distance IS 'Distance from closest landmark';
+COMMENT ON COLUMN water_body.water_body_name IS 'Common or popular name of the water-body';
+COMMENT ON COLUMN water_body.water_body_latitude IS 'Latitude of water body location';
+COMMENT ON COLUMN water_body.water_body_longitude IS 'Longitude of water body location';
+COMMENT ON COLUMN water_body.country_code IS 'Country of the water-body location. Joint foreign key reference to country_province table country_code column along with province_code.';
+COMMENT ON COLUMN water_body.province_code IS 'Province of the water-body location. Joint foreign key reference to country_province table province_code column along with country code.';
+COMMENT ON COLUMN water_body.closest_city IS 'Nearest city/landmark from the water body';
+COMMENT ON COLUMN water_body.distance IS 'Distance from closest city/landmark';
```

Comment:
Is "distance" measured in Kilometers? Miles? Both? Either way, the column comment should specify so it's clear to anyone using the column.

Reply:
Resolved